entity-framework/ef6/fundamentals/testing/testability-article.md

```diff
@@ -958,10 +958,10 @@ In this paper we’ve demonstrated several approaches to creating testable code
 -   Robert C. Martin, “ [The Single Responsibility Principle](https://www.objectmentor.com/resources/articles/srp.pdf)”
 -   Martin Fowler, [Catalog of Patterns](https://www.martinfowler.com/eaaCatalog/index.html) from *Patterns of Enterprise Application Architecture*
 -   Griffin Caprio, “ [Dependency Injection](https://msdn.microsoft.com/magazine/cc163739.aspx)”
--   Data Programmability Blog, “ [Walkthrough: Test Driven Development with the Entity Framework 4.0](https://blogs.msdn.com/adonet/pages/walkthrough-test-driven-development-with-the-entity-framework-4-0.aspx)”.
--   Data Programmability Blog, “ [Using Repository and Unit of Work patterns with Entity Framework 4.0](https://blogs.msdn.com/adonet/archive/2009/06/16/using-repository-and-unit-of-work-patterns-with-entity-framework-4-0.aspx)”
+-   Data Programmability Blog, “ [Walkthrough: Test Driven Development with the Entity Framework 4.0](https://docs.microsoft.com/en-us/archive/blogs/adonet/walkthrough-test-driven-development-with-the-entity-framework-4-0)”.
```

Comment:
This causes warning due to en-us being in link.
See https://github.com/dotnet/EntityFramework.Docs/issues/1900

Reply:
Would you like a new PR?

Reply:
PR here: https://github.com/dotnet/EntityFramework.Docs/pull/2264